mne/event.py

```diff
@@ -827,3 +828,289 @@ def concatenate_events(events, first_samps, last_samps):
         events_out = np.concatenate((events_out, e2), axis=0)
 
     return events_out
+
+
+class Elekta_event(object):
```

Comment:
I think all these classes should be private if possible -- people should never need to see them if we can avoid it. But maybe I'm missing some future API design plans.